### PR TITLE
app-arch/innoextract: Bump to version 1.8

### PR DESCRIPTION
**Pull request by dscharrer**

### CODE REVIEW

File: app-arch/innoextract/innoextract-1.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-arch/innoextract/innoextract-1.8.ebuild]: 7 should work here.

File: app-arch/innoextract/innoextract-1.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+inherit cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
```
Comment (juippis) [inline app-arch/innoextract/innoextract-1.8.ebuild]: `https://` seems to be available.

File: app-arch/innoextract/innoextract-1.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
```
Comment (juippis) [inline app-arch/innoextract/innoextract-1.8.ebuild]: Similarily here.

File: app-arch/innoextract/innoextract-1.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +iconv +lzma"
+
+RDEPEND="
+	dev-libs/boost:=
+	iconv? ( virtual/libiconv )
+	lzma? ( app-arch/xz-utils )"
+DEPEND="${RDEPEND}"
+
+DOCS=( README.md CHANGELOG )
```
Comment (juippis) [inline app-arch/innoextract/innoextract-1.8.ebuild]: These are part of default `einstalldocs` array, so you can remove this part and they will still be installed.

File: app-arch/innoextract/innoextract-1.8.ebuild (hunk 1 above)
Comment (dscharrer) [reply app-arch/innoextract/innoextract-1.8.ebuild]: I have updated the PR with the suggested changes.

**Pull request closed without merging**